src/Plugins/SimplnxCore/wrapping/python/simplnxpy.cpp

```diff
@@ -453,6 +453,57 @@ PYBIND11_MODULE(simplnx, mod)
   dataType.value("float64", DataType::float64);
   dataType.value("boolean", DataType::boolean);
 
+  mod.def(
+      "numpy_dtype_to_datatype",
```

Comment:
```suggestion
      "convert_np_dtype_to_datatype",
```
I think adding "convert" helps make it clear that this is a function. And shortening "numpy" to "np" helps to keep it from getting too long.